Sat Exercise (1).py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[17]:
+
+
+import seaborn as sns
+import numpy as np
+import pandas as pd
+import datetime
+
+
+# In[2]:
+
+
+get_ipython().run_line_magic('matplotlib', 'inline')
+
+
+# In[6]:
+
+
+file=r"D:\Sukreet\Training\Material\Sat exercise\bike_sharing_hours.csv"
+data=pd.read_csv(file)
+
+
+# In[26]:
+
+
+data
+
+
+# In[ ]:
+
+
+#1.1
+
+
+# In[35]:
+
+
+data.dteday = pd.to_datetime(data.dteday)
+
+
+# In[47]:
+
+
+data.iloc[data['dteday'].day<7]
+
+
+# In[5]:
+
+
+from scipy.stats import levene
+
+
+# In[43]:
+
+
+levene(data.loc[(data['mnth']<7)&(data['dteday'].day<=7),'cnt'],data.loc[(data['mnth']>=7)&(data['dteday'].day<=7),'cnt'])
+
+
+# In[ ]:
+
+
+# this means variances are equal
+
+
+# In[10]:
+
+
+from scipy.stats import ttest_ind
+
+
+# In[12]:
+
+
+ttest_ind(data.loc[(data['mnth']<7)&(data['workingday']<=7),'cnt'],
+          data.loc[(data['mnth']>=7)&(data['workingday']<=7),'cnt'],
+          equal_var=True)
+
+
+# In[ ]:
+
+
```

Comment:
whats you conclusion ?

Reply:
Hi Lalit Please refer to Sat Exercise (2).py
I had deleted this file. This was an initial commit.